LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2016-2017 Rob Wu rob@robwu.nl
```

Comment:
Can you put angle brackets around the e-mail address?

And please edit the commit description to explicitly call this the MIT license.